src/widgets/_lists.scss

```
@@ -0,0 +1,23 @@
+list {
+    background-color: bg-color(1);
+
+    popover & {
+        background-color: transparent;
+    }
+
+    row {
+        &:selected {
+            background-color: rgba($fg-color, 0.15);
```

Okay this is a tiny change but it feels closer to the feel of the highlight colors in general. Thoughts?

```suggestion
            background-color: rgba($fg-color, 0.1);
```